packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -132,13 +132,15 @@ The test account allows the OIN team to verify that your integration flow works
 
 A configuration guide helps your customers understand how to configure your Okta integration to work with your cloud application.
 
-Provide a configuration guide as part of the OIN submission process for all integrations:
+Provide a configuration guide as part of the OIN submission process for each type of integration:
```

Comment:
@vanngo-okta Can we replace it with:
"Provide a separate configuration guide as part of the OIN submission process for each type of integration"

Reply:
Agree with @ykrestyanov-okta.